Form-Controls/index.html

```diff
@@ -13,15 +13,67 @@ <h1>Product Pick</h1>
     </header>
     <main>
       <form>
+        <section>
+          <fieldset>
+  <h2>Customer information</h2>
+     <p>
+    <label for="name">
+      <span>Full Name: </span>
+    </label>
+    <input type="text" id="name" name="username" required pattern="[A-Za-z\s]{2,}"
+              title="Please enter a valid name (at least 2 letters)."/>
+  </p>
+  <p>
+    <label for="mail">
+      <span>Email: </span>
+    </label>
+    <input type="email" id="mail" name="user-mail" required />
+  </p>
+<fieldset>
+  <h2>Product specification</h2>
+  <p>
+    <label for="size">
+      <span>Select a size:</span>
+    </label>
+    <select id="size" name="product size">
+      <option value=""></option>
+      <option value="XS">XSmall</option>
+      <option value="S">Small</option>
+      <option value="M">Medium</option>
+      <option value="L">Large</option>
+      <option value="XL">XLarge</option>
+      <option value="XXL">XXLarge</option>
+      </select>
+  </p>
+  <p>
+    <label for="Colour">
+      <span>Select a colour:</span>
+    </label>
+    <select id="Colour" name="product colour">
+      <option value=""></option>
```

Comment:
In order to make your select field more meaningful you can make add "Choose" label to first option and add 
`disabled hidden selected` attrribute to option tag.